librpc/Rpc.cpp

```diff
@@ -1006,7 +1006,8 @@ std::string Rpc::sendRawTransaction(int _groupID, const std::string& _rlp)
         if (currentTransactionCallback)
         {
             auto transactionCallback = *currentTransactionCallback;
-            tx.setRpcCallback([transactionCallback](LocalisedTransactionReceipt::Ptr receipt) {
+            tx.setRpcCallback([transactionCallback](
+                                  LocalisedTransactionReceipt::Ptr receipt, bytes const& input) {
```

Comment:
```suggestion
                                  LocalisedTransactionReceipt::Ptr receipt, bytes input) {
```

Reply:
@bxq2011hust why do you think can not use const& here ?

Reply:
> @bxq2011hust why do you think can not use const& here ?
because this is async call

Reply:
Ok, I removed the const& for the input parameter.